libkbfs/keybase_service_base.go

```diff
@@ -741,6 +772,22 @@ func (k *KeybaseServiceBase) FSSyncStatusRequest(ctx context.Context,
 	return k.kbfsClient.FSSyncStatus(ctx, resp)
 }
 
+// TeamChanged implements keybase1.NotifyTeamInterface for
+// KeybaseServiceBase.
+func (k *KeybaseServiceBase) TeamChanged(
+	ctx context.Context, arg keybase1.TeamChangedArg) error {
+	k.log.CDebugf(ctx, "Flushing cache for team %s/%s "+
+		"(membershipChange=%t, keyRotated=%t, renamed=%t)",
+		arg.TeamName, arg.TeamID, arg.Changes.MembershipChanged,
+		arg.Changes.KeyRotated, arg.Changes.Renamed)
+	k.setCachedTeamInfo(arg.TeamID, TeamInfo{})
+
+	if arg.Changes.Renamed {
```

Comment:
Unrelated to this PR, but is this also how we get notified when user is changed from writer to reader, and thus know to stop pushing any outstanding MD writes to `mdserver`?